contrib/redis/rueidis/example_test.go

```diff
@@ -0,0 +1,35 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
```

Comment:
Super small nit, but for new files, make sure the copyright year is correct (i.e. 2025).